### DISCUSSION TITLE
Running time response

### DISCUSSION THREAD

**Discussion by ahmadijo** [petrobras/ross#1033] [Q&A]
Hi
I am exactly following your code in Sections 1.6.1 and 1.6.2 of the tutorial, but the result is different. Actually, both one and two probes have the same phase angle.

> speed = 600.0
node = 26
F = np.zeros((time_samples, rotor3.ndof))
component on direction x
F[:, 4 * node + 0] = 10 * np.cos(2 * t)
component on direction y
F[:, 4 * node + 1] = 10 * np.sin(2 * t)
response3 = rotor3.run_time_response(speed, F, t)

![newplot](https://github.com/petrobras/ross/assets/72427100/f07f9a6d-7325-4170-8a06-0b23df2f74d7)

Do you know what is the issue?

**Comment by raphaeltimbo** [comment]
Hi @ahmadijo!
It seems that a modification recently introduced in the code is causing this.
We are evaluating it in this issue #999.
Thank you for bringing that up.
